easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,39 @@
+name = 'netCDF'
+version = '4.4.1.1'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.unidata.ucar.edu/downloads/netcdf/ftp/']
+
+dependencies = [
+    ('cray-hdf5/1.10.0', EXTERNAL_MODULE),
+]
+
+postinstallcmds = [('ln -s %(installdir)s/lib64 %(installdir)s/lib')]
+
+builddependencies = [
+    ('CMake', '3.9.1', '', True),
+    #('Doxygen', '1.8.9.1', '', True),
+    ('cURL', '7.49.1', '', True),
```

Comment:
`Irresolvable dependencies encountered: cURL/.7.49.1` => need to push it

Reply:
https://github.com/eth-cscs/production/blob/craype/17.08/easybuild/easyconfigs/c/cURL/cURL-7.55.1-CrayGNU-17.08.eb

Reply:
Done.